vagrant-post-script/orchestrate_docker.sh

```diff
@@ -3,13 +3,12 @@
 cd /vagrant/ || exit 1
 
 if [[ -f .env ]]; then
-  dos2unix .env
+  chmod +x helper.sh
+  dos2unix .env Makefile helper.sh
   make start
 else
   cp .env.example .env
-
-  #git reset --hard HEAD
-  #./helper.sh --bumpmodules
+  chmod +x helper.sh
+  dos2unix .env Makefile helper.sh
   make bootstrap
-  sleep 120 #to give time for the bootstrap
```

Comment:
we can also use `./helper.sh --wait-until-healthy` instead of it somewhere in the futuer

Reply:
i realised that the CLI is not responding anyway till it's finished (so, the message `finally` only appears afterwards anyway)

Reply:
basically it was only a way to not make people go on the browser till time had passed, it is not technical